### PR TITLE
Adding Common 0.7.1

### PR DESCRIPTION
**Pull request by jamdavi**
Adding a reference to DotNetty.Common 0.7.1 to fix the threading issue introduced in 0.7.0.

Ref: #2163 #2194

### CODE REVIEW

File: iothub/device/src/Microsoft.Azure.Devices.Client.csproj [hunk 1]
```diff
@@ -79,6 +79,7 @@
     <Compile Include="ModernDotNet\HsmAuthentication\Transport\HttpRequestResponseSerializer.cs" />
     <Compile Include="ModernDotNet\HsmAuthentication\Transport\UnixDomainSocketEndPoint.cs" />
     <PackageReference Include="DotNetty.Codecs.Mqtt" Version="0.7.0" />
+    <PackageReference Include="DotNetty.Common" Version="0.7.1" />
```
Comment (drwill-ms) [inline iothub/device/src/Microsoft.Azure.Devices.Client.csproj:82]: Do we want to consider adding a comment that this is temporarily needed until DotNetty.Codecs.Mqtt updates its reference?

**Review by drwill-ms — APPROVED** [review]

File: iothub/device/tests/Microsoft.Azure.Devices.Client.Tests.csproj [hunk 2]
```diff
@@ -26,6 +26,7 @@
     <Compile Remove="ClientWebSocketTransportTests.cs" />
     <Compile Remove="Mqtt\ClientWebSocketChannelTests.cs" />
     <PackageReference Include="DotNetty.Codecs.Mqtt" Version="0.7.0" />
+    <PackageReference Include="DotNetty.Common" Version="0.7.1" />
```
Comment (abhipsaMisra) [inline iothub/device/tests/Microsoft.Azure.Devices.Client.Tests.csproj:29]: q - with the test project having a reference to the source code project, don't we get all these dependencies automatically?

**Review by abhipsaMisra — APPROVED** [review]

**Review by azabbasi — APPROVED** [review]

File: iothub/device/tests/Microsoft.Azure.Devices.Client.Tests.csproj (hunk 2 above)
Comment (jamdavi) [reply iothub/device/tests/Microsoft.Azure.Devices.Client.Tests.csproj:29]: I did this blindly by adding references to any assemblies that referenced DotNetty. I'm not 100% sure if the transitive dependencies overwrite the specific ones from the referenced package.

**Pull request merged**